tests/integration/profiling/rule_based_profilers/test_run_profiler_notebook.py

```diff
@@ -148,21 +155,12 @@ def test_run_self_initializing_expectations_notebook(tmp_path):
         with open(output_notebook_path, mode="w", encoding="utf-8") as f:
             nbformat.write(nb, f)
         # clean up Expectations directory after running test
-        try:
-            shutil.rmtree(os.path.join(base_dir, "great_expectations/expectations/tmp"))
-            os.remove(
-                os.path.join(
-                    base_dir, "great_expectations/expectations/.ge_store_backend_id"
-                )
-            )
-            os.remove(
-                os.path.join(
-                    base_dir,
-                    "great_expectations/expectations/new_expectation_suite.json",
-                )
-            )
-        except FileNotFoundError:
-            logger.debug(
-                "Files were already deleted by running the optional last cell in the notebook. "
-                "We therefore allow the test to pass"
+
+    clean_up_test_files(
+        base_dir=base_dir,
+        additional_files_to_clean_up=[
+            os.path.join(
+                base_dir, "great_expectations/expectations/new_expectation_suite.json"
             )
+        ],
```

Comment:
If sticking with this current API, can we do this `join` in the actual method? We have access to the `base_dir` through the args right?

Reply:
done!